### DISCUSSION TITLE
Welcome to event-sep-2021 Discussions!

### DISCUSSION THREAD

**Discussion by lugti** [lugti/event-sep-2021#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as our primary medium to connect and discuss,
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other society members

 Do acknowledge this, either by reacting with an emoji or dropping a comment below.

**Comment by jayantkatia** [comment]
@kartikmadan26 @Sahaj-Srivastava24  @simranbhalla3
